### DISCUSSION TITLE
Welcome to Data-mining Discussions!

### DISCUSSION THREAD

**Discussion by nathalylorena** [nathalylorena/Data-mining#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iurysantos-hub** [comment]
you use colab google @nathalylorena

**Reply by nathalylorena** [reply]
yes i use @iurysantos-hub

**Comment by marylorh** [comment]
studies data science

**Comment by alikhazaeii** [comment]
Good👍👍

**Comment by Aditya-Mishra799** [comment]
I also want to join this community. I have experience with software development especially in field of WEB. Can you please tell me what the goals of this community are ? Is it related to Data science.
